### DISCUSSION TITLE
How to configure MSEide under Windows

### DISCUSSION THREAD

**Discussion by alas82** [mse-org/mseide-msegui#41] [Q&A] [answered]
hello , thanks for providing this IDE
could you please make detailed video how to configure it and do first example, or either provide setup file can configure it automatically 
i have tried doing this 100 of time and never worked 

thanks

**Comment by fredvs** [comment]
Hello and welcome to mseide-msegui discussions.

About how to install mseide-msegui there are explanation in the readme.md file.
https://github.com/mse-org/mseide-msegui/blob/main/readme.md

Here a video of the installation + load-run the demo ( fpc compiler has to be installed ).

https://user-images.githubusercontent.com/3421249/198116838-e3eaa3ca-0150-4df5-8c9f-b23bd945f5ba.mp4

**Reply by alas82** [reply]
hi , i followed your video steps , as per capture.png, 
![Capture](https://user-images.githubusercontent.com/65008114/199085969-5dab39cd-fd42-4b9d-935f-a907f539d5cf.PNG)

i got this result 

![Capture1](https://user-images.githubusercontent.com/65008114/199085980-401cb1a5-d736-474c-a94c-965a4a321726.PNG)

can you please help
thanks

**Comment by fredvs** [comment, marked as answer]
Hello.

It is because the gdb.exe debugger seems not be installed or the path is not found.
https://www.sourceware.org/gdb/

I dont have here a Windows machine to test how to install gdb.exe.
If you have Lazarus  installed, it has gdb.exe in C:\lazarus\mingw\..\bin\gdb.exe and you may use that full path in settings.
I think also that in FPC package gdb.exe is provided.

Otherwise if you only want to compile the project, go menu Project/Build and do not click on the green button.
If you have a error of fpc.exe not found, you have to enter in settings the full path of fpc.exe.

Hope it helps.

Also for Windows, the compiler is **fpc.exe**.
Did you install the fpc compiler?
